Library/Homebrew/caveats.rb

```diff
@@ -209,10 +197,29 @@ def plist_caveats
         s << "  #{f.plist_manual}"
       end
 
+      # pbpaste is a system clipboard tool for OSX that doesn't work well with tmux
+      # This checks if tmux is being used and warns about brew failure
       if ENV["TMUX"] && !quiet_system("/usr/bin/pbpaste")
         s << "" << "WARNING: brew services will fail when run under tmux."
       end
     end
     s.join("\n") + "\n" unless s.empty?
   end
+
+  def plist_complete_path
+    destination = if f.plist_startup
+      "/Library/LaunchDaemons"
+    else
+      "~/Library/LaunchAgents"
+    end
+
+    plist_filename = if f.plist
+      f.plist_path.basename
+    else
+      File.basename Dir["#{keg}/*.plist"].first
+    end
+    destination_path = Pathname.new File.expand_path destination
```

Comment:
`destination_path = Pathname.new(File.expand_path(destination))` makes this a bit easier to follow